tests/test_snql_api.py

```diff
@@ -1333,6 +1333,38 @@ def test_tags_key_column(self) -> None:
 
         assert response.status_code == 200
 
+    def test_tags_column_in_join(self) -> None:
+        response = self.post(
+            "/events/snql",
+            data=json.dumps(
+                {
+                    "dataset": "events",
+                    "query": f"""MATCH (events: events) -[attributes]-> (ga: group_attributes)
+                        SELECT count() AS `count`
+                        BY events.time
+                        WHERE events.timestamp >= toDateTime('{self.base_time.isoformat()}')
+                        AND ifNull(events.tags[service-class], '') != 'devtest'
```

Comment:
so in this case service-class is a column on the entity?

Reply:
No, `tags` is a column on the entity.

Reply:
ah, got it